### DISCUSSION TITLE
SNAPSHOT-20221031.204527

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#981] [General]
* Neovim 0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/snapshot/20221031.204527'>SNAPSHOT-20221031.204527</a>.</em>

**Comment by kemiller** [comment]
Hey, when I tried this, it reports Neovim version 0.9.0-dev, which is creating some incompatibilities.  Can we get a pure 0.8.0 cut?
